searcch_backend/api/resources/artifact_request.py

```diff
@@ -179,8 +179,14 @@ def post(self, artifact_group_id, artifact_id=None):
             agreement_file = request.files.get('file')
             if not agreement_file:
                 abort(400, description="missing agreement file")
-            
             agreement_file = agreement_file.read()
+
+            irb_file = request.files.get('irb_file')
+            if not irb_file:
+                irb_file = ""
```

Comment:
any reason we can't make this a None instead and allow a NULL in the database to indicate that an IRB didn't exist?  that's generally safer than a blank string.

Reply:
Resolved in commit: 3190e2ad52525bf82d65698ee2058e238b623deb